evalml/tests/component_tests/test_datetime_featurizer.py

```diff
@@ -139,3 +147,33 @@ def test_datetime_featurizer_numpy_array_input():
     assert list(datetime_transformer.transform(X).columns) == ["0_year", "0_month", "0_day_of_week", "0_hour"]
     assert datetime_transformer.get_feature_names() == {'0_month': {'February': 1, 'June': 5, 'May': 4},
                                                         '0_day_of_week': {'Saturday': 6, 'Tuesday': 2}}
+
+
+@pytest.mark.parametrize("X_df", [pd.DataFrame(pd.to_datetime(['20190902', '20200519', '20190607'], format='%Y%m%d')),
+                                  pd.DataFrame(pd.Series([1, 2, 3], dtype="Int64")),
+                                  pd.DataFrame(pd.Series([1., 2., 3.], dtype="float")),
+                                  pd.DataFrame(pd.Series(['a', 'b', 'a'], dtype="category")),
+                                  pd.DataFrame(pd.Series([True, False, True], dtype="boolean")),
+                                  pd.DataFrame(pd.Series(['this will be a natural language column because length', 'yay', 'hay'], dtype="string"))])
+@pytest.mark.parametrize("with_datetime_col", [True, False])
+def test_datetime_featurizer_woodwork_custom_overrides_returned_by_components(X_df, with_datetime_col):
```

Comment:
Can you add coverage for `encode_as_categories=True`?

Reply:
Yes, thank you for calling this out!